tests/flows/native_token_bridge.go

```diff
@@ -114,6 +114,8 @@ func NativeTokenBridge(network interfaces.LocalNetwork) {
 			sourceSubnet.BlockchainID,
 			bridgeContractAddress,
 			initialReserveImbalance,
+			big.NewInt(1),
```

Comment:
A multiplier > 1 would be good to test in the e2e flow.

Reply:
Since this feature is completely contained to `NativeTokenDestination`, I think it's probably fine for it to be tested by the unit tests, and leave the e2e tests to focus on the cross-chain interactions.

Reply:
Yeah, that's fair, though the e2e tests definitely should test the contract-specific features as well as cross-chain messaging. I think in general, we test basic contract functionality in the e2e tests, then dive deeper into edge cases in the unit tests.